_topics/en/side-channels.md

```diff
@@ -44,14 +50,19 @@ optech_mentions:
 # see_also:
 #   - title:
 #     link:
+---
+Side channel attacks can be generally categorized as
+- timing attacks
+- power-monitoring attacks
+- acoustic cryptanalysis
+- electromagnetic attacks
+- cache attacks
 
-## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
-## Should be less than 500 characters
-excerpt: >
-  **Side channels** are weaknesses in security protocols that arise from
-  flaws in the hardware and software used to implement the protocol,
-  rather than from flaws in the protocol's algorithms.
+Statistical methods and machine learning algorithms are often used in combination with physical access to a hardware signer in order to carry out attacks. For example, an attacker can take a factory hardware signer, gather a dataset that measures changes in power consumption during critical calculations on this signer, then use this dataset on a new signer, measure this new signer's power consumption during its critical calculations, and exfiltrate the passcode by comparing the power consumption of the new signer with the statistical analysis done on the factory signer.
+
+Some side channel attack surfaces in the Bitcoin ecosystem have been mitigated or precluded entirely. For example, the libsecp256k1 library has merged the SafeGCD algorithm into its codebase. The SafeGCD algorithm is a way to calculate modular inverse operations in constant time. A modular inverse operation is a logical operation used when verifying ECDSA signatures.
+
+TROPIC01 is an auditable RISCV secure chip under development, with the intention to release it as part of a hardware signer. Using secure chips may mitigate some side channel attacks that hardware signers are currently vulnerable to.
```

Comment:
I think this first sentence is too specific about something we haven't discussed in the newsletter.  I think it's ok to drop it and just use the second sentence here.

Reply:
@paulkania if you agree with this suggestion, would you mind making the change?

Reply:
@paulkania going to close this for now until we resolve this suggestion. feel free to re-open this PR!